engines/wasmtime/build.rs

```diff
@@ -66,7 +66,7 @@ fn main() {
     // Build the engine library.
     section("Building the engine");
     exec(
-        &["cargo", "build", "--release", "-p", "wasmtime-bench-api"],
+        &["cargo", "build", "--release", "-p", "wasmtime-bench-api", "--features", "wasi-nn"],
```

Comment:
Did we figure out if this was actually necessary? I think it would be nicer (but this is a nit, not a real complained) if it could be removed either a) because it is unnecessary or b) by an upstream change to build in wasi-nn by default to `bench-api`.

Reply:
Yes this isn't needed, I'll take it out.